adminSiteClient/gdocsValidation.ts

```diff
@@ -146,18 +115,24 @@ export class ContentHandler extends AbstractHandler {
 // honor the type cast (and subsequent assumptions) in getPublishedGdocs()
 export const getErrors = (gdoc: OwidArticleType): ErrorMessage[] => {
     const errors: ErrorMessage[] = []
+    const attachmentErrors: ErrorMessage[] =
```

Comment:
TODO: create these as `ErrorMessage`s in the server instead, and make "linking to an unpublished document" a warning instead of an error, so that authors can still publish through it if they want.